osu.Game/Overlays/NowPlayingOverlay.cs

```diff
@@ -278,12 +294,34 @@ protected override void Update()
                 progressBar.EndTime = track.Length;
                 progressBar.CurrentTime = track.CurrentTime;
 
+                // Process the Current Minute and Second
+                CurrentMinutes = Math.Floor((track.CurrentTime / 1000) / 60).ToString();
+                CurrentSeconds = Math.Floor((track.CurrentTime / 1000) % 60).ToString();
+
+                // Process the Total Minute and Second
+                FinalMinutes = Math.Floor((track.Length / 1000) / 60).ToString();
+                FinalSeconds = Math.Floor((track.Length / 1000) % 60).ToString();
+
+                // If the second variables is only 1 Character long, it will be made 2
+                if (CurrentSeconds.Length < 2)
+                {
+                    CurrentSeconds = "0" + CurrentSeconds;
+                };
+
+                if (FinalSeconds.Length < 2)
+                {
+                    FinalSeconds = "0" + FinalSeconds;
+                };
```

Comment:
for consistent style
```suggestion
                // If the second variables is only 1 Character long, it will be made 2
                if (CurrentSeconds.Length < 2)
                    CurrentSeconds = "0" + CurrentSeconds;

                if (FinalSeconds.Length < 2)
                    FinalSeconds = "0" + FinalSeconds;
```
I feel like this is somethign that can be better achieved with string formatting. Not too sure how it works in C#, though.